src/smc-hub/postgres-server-queries.coffee

```diff
@@ -221,6 +227,9 @@ exports.extend_PostgreSQL = (ext) -> class PostgreSQL extends ext
                         'value::TEXT' : (new Date()).toISOString()
                     conflict : 'name'
                     cb     : cb
+            # and clear the cache
+            (cb) ->
+                SERVER_SETTINGS_CACHE.reset()
```

Comment:
You forgot to call cb here.  And clearing the cache probably isn't async anyways, so this should just be done if there isn't an error right before line 234 below.   

It is WAY too hard to write callback-hell code, and I can't wait until we convert all this to async/await (+typescript), since that would have made **precisely** this bug simply not happen!

Reply:
I'll fix this, but please ack that you saw it.